src/nifake/custom_types/custom_struct.py

```diff
@@ -0,0 +1,33 @@
+import ctypes
+
+from nifake import visatype
+
+
+class CustomStruct(ctypes.Structure):
+    _fields_ = [
+        ('struct_int', visatype.ViInt32),
+        ('struct_double', visatype.ViReal64),
+    ]
+
+    def __init__(self, data=None, struct_int=0, struct_double=0.0):
+        super(ctypes.Structure, self).__init__()
+        if data is not None:
+            self.struct_int = data.struct_int
+            self.struct_double = data.struct_double
+        else:
+            self.struct_int = struct_int
+            self.struct_double = struct_double
+
+
+class CustomStructPython(object):
+    def __init__(self, data=None, struct_int=0, struct_double=0.0):
+        if data is not None:
+            self.struct_int = data.struct_int
+            self.struct_double = data.struct_double
+        else:
+            self.struct_int = struct_int
+            self.struct_double = struct_double
+
+
+custom_struct = CustomStructPython
```

Comment:
This is strange... I think it's ok for the customer to reference CustomStruct (no Python) (uppercase, camelCase) because it is a type. As far as our convention goes, custom_struct is not the name of a type.

I think the type CustomStruct(ctypes.Structure) should be renamed to _CustomStruct or CustomStructCType or something because it is private.

Reply:
See above